README.md

```diff
@@ -27,6 +27,13 @@ Navigate to the root of the project then run the tests from the command line
 admin$ rspec
 ```
 
+## How to analyse the complexity of the commits
+Navigate to the root of the project then run the tests from the command line
+
+```console
+admin$ rubycritic app app/controllers/application_controller.rb
+```
```

Comment:
might be nice to put an example of the output from ruby critic so we know what it does? (coming from someone who has no idea!!!)

Reply:
I love the idea. Will do it now!

Reply:
When would we need to run this command? Before/after every commit? Pull request?

Reply:
Done.